docs/synapse/devops.rst

```diff
@@ -17,13 +17,83 @@ Docker Deployment
 Configuring A Mirror
 --------------------
 
-Configuring A Remote Layer
---------------------------
-
 Axon
 ====
 
 Cryotank
 ========
 
+Migration
```

Comment:
This section should be within the Cortex heading and should be named "0.1.x -> 0.2.0 Migration" or something :)